index.bs

```diff
@@ -241,6 +243,66 @@ The <dfn attribute for=FileSystemHandle>name</dfn> getter steps are to return [=
   :: Returns true if |handle1| and |handle2| represent the same file or directory.
 </div>
 
+### The {{FileSystemHandle/remove()}} method ### {#api-filesystemhandle-remove}
+
+<div class="note domintro">
+  : await |handle| . {{FileSystemHandle/remove()|remove}}()
+  : await |handle| . {{FileSystemHandle/remove()|remove}}({ {{FileSystemRemoveOptions/recursive}}: false })
+  :: Attempts to remove the entry represented by |handle| from the underlying file system.
+
+     Attempting to delete a file or directory that does not exist is considered success,
+     while attempting to delete a non-empty directory will result in a promise rejection.
+
+  : await |handle| . {{FileSystemHandle/remove()|remove}}({ {{FileSystemRemoveOptions/recursive}}: true })
+  :: Attempts to remove the entry represented by |handle| from the underlying file system.
+
+     If the entry is a directory, its contents will also be deleted recursively.
+
+     Attempting to delete a file or directory that does not exist is considered success.
+
+     For files or directories with multiple hardlinks or symlinks, the entry which is deleted
+     is the entry corresponding to the path which was used to obtain the handle.
+</div>
+
+<div algorithm>
+The <dfn method for=FileSystemHandle>remove(|options|)</dfn> method steps are:
+
+1. Let |result| be [=a new promise=].
+1. Run these steps [=in parallel=]:
```

Comment:
Do these steps happen in parallel, so the lock taking races with other `remove()` calls? Or should we really have some kind of "parallel queue" here? (See HTML.)

Reply:
Hmm that's a good point. I believe `removeEntry()`, `move()`, and `createSyncAccessHandle()` all have the same issue? Locking should probably not race with other operations which take a lock.

Reply:
Opened #74 for this